unittests/CppInterOp/CUDATest.cpp

```diff
@@ -30,7 +30,7 @@ static bool HasCudaRuntime() {
     if (!HasCudaSDK())
       return false;
 
-    Cpp::CreateInterpreter({}, {"--cuda"});
+    if(!Cpp::CreateInterpreter({}, {"--cuda"})) return false;
```

Comment:
```suggestion
    if (!Cpp::CreateInterpreter({}, {"--cuda"})) 
      return false;
```

why clang-format was happy with this? Please fix the other occurrences too.

Reply:
Oops completely missed this (not sure I saw a complaint regarding this !

Shall address it, thanks